DevoxxClientMobile/src/main/java/com/devoxx/service/DevoxxService.java

```diff
@@ -251,6 +264,10 @@ public DevoxxService() {
         });
     }
 
+    private boolean isConferenceFromPast(Conference conference) {
+        return conference.getDaysUntilEnd() < 0;
```

Comment:
Is there an advantage of introducing a transition period?

Reply:
No, it was just a comment, no obligation. I was just thinking that it looks odd that you would see a notification message that you need to close when the conference is just over. After a week, it would make more sense, as most will have left the conference mentally as well by then 😄.

Reply:
I have adding a custom variable in DevoxxSettings which can be used to change this as and if required.